### PR TITLE
Added Feature requested #1534

### PR DESCRIPTION
**Pull request by Hassibayub**
# Motivation
Added requested feature at issue #1534

### CODE REVIEW

**Comment by Hassibayub** [comment]
Why is this branch still pending, and how long would it take to merge into the main? @aandergr

File: instaloader/__main__.py [hunk 1]
```diff
@@ -15,6 +15,31 @@
...
+    """Take the list of targets and remove
+    - Blanklines
+    - lines starts withs #
+    - # from line, used as line comment
+
+    Args:
+        targets (List[str]): List contains args strings
+
+    Returns:
+        List[str]: Clean list
```
Comment (aandergr) [inline instaloader/__main__.py:28]: This is not the right syntax for our docstring; see https://www.sphinx-doc.org/en/master/tutorial/describing-code.html or the other docstrings in Instaloader for a reference of the syntax.

File: instaloader/__main__.py [hunk 2]
```diff
@@ -128,6 +153,7 @@ def _main(instaloader: Instaloader, targetlist: List[str],
     anonymous_retry_profiles = set()
     try:
         # Generate set of profiles, already downloading non-profile targets
+        targetlist = targetlist_parser(targetlist) # Removes Blanklinks
```
Comment (aandergr) [inline instaloader/__main__.py:156]: This does not seem clean to me. What is the advantage of changing the parsed targets, rather than overriding [ArgumentParser.convert_arg_line_to_args()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.convert_arg_line_to_args)?

File: instaloader/__main__.py [hunk 3]
```diff
@@ -250,7 +276,8 @@ def main():
     parser = ArgumentParser(description=__doc__, add_help=False, usage=usage_string(),
                             epilog="The complete documentation can be found at "
                                    "https://instaloader.github.io/.",
-                            fromfile_prefix_chars='+')
+                            fromfile_prefix_chars='+'
+                            )
```
Comment (aandergr) [inline instaloader/__main__.py:280]:
```suggestion
                            fromfile_prefix_chars='+')
```

(revert this unnecessary change)

File: instaloader/__main__.py [hunk 4]
```diff
@@ -15,6 +15,31 @@
...
+
+    Returns:
+        List[str]: Clean list
+    """
+
+    parsed_targets = []
+
+    for target in targets:
+        # Ignore blank lines and line startswith
+        if len(target) <= 0 or target.isspace() or target.strip().startswith("#"):
```
Comment (aandergr) [inline instaloader/__main__.py:35]: This breaks hashtags.

**Comment by Hassibayub** [comment]
I will clear these issues
